Tools/AP_Periph/esc_apd_telem.cpp

```diff
@@ -46,6 +46,10 @@ bool ESC_APD_Telem::update() {
                 // valid stop byte, check the CRC
                 if (crc_fletcher16(received.bytes, 18) == received.packet.checksum) {
                     // valid packet, copy the data we need and reset length
+                    // SKYWAYS start
+                    // There's no ESC status dronecan field, so hijacking error_count
+                    decoded.error_count = le32toh(received.packet.status_flags);
```

Comment:
Won't you need to create a `le8toh` here? https://github.com/brha8286/ardupilot/blob/ac42f5fb5ebf7e6daaaa8b5964b7f8c13fa43f19/libraries/AP_HAL/utility/sparse-endian.h#L84

Reply:
And then I think you're going to have problems packing that into a `uint32`? Do we have to hijack the message like this?